c7n/resources/hsm.py

```diff
@@ -17,64 +20,37 @@ class resource_type(TypeInfo):
         id = name = 'ClusterId'
         filter_name = 'Filters'
         filter_type = 'scalar'
-        # universal_taggable = True
-        # Note: resourcegroupstaggingapi still points to hsm-classic
+        universal_taggable = True
```

Comment:
Would it make sense to use `object()` here instead? I don't think that we need the compatibility on it since there was never a tag-count, mark, unmark or untag action available.